swagger/messaging/activity_message.json

```diff
@@ -0,0 +1,249 @@
+          },
+          "errors": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string"
+                },
+                "message": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "required": [
+          "success",
+          "error"
+        ]
+      },
+      "RequestDto": {
+        "type": "object",
+        "properties": {
+          "business_uid": {
+            "type": "string",
+            "description": "The business that is sending the message."
+          },
+          "staff_uid": {
+            "type": "string",
+            "description": "The staff that is sending the message."
+          },
+          "client_uid": {
+            "type": "string",
+            "description": "Client to send the message to."
+          },
+          "message_uid": {
+            "type": "string",
+            "description": "The entity's unique identifier."
+          },
+          "activity_type": {
+            "type": "string",
+            "enum": [
+              "invite"
+            ],
+            "description": "Activity category type from a closed list."
+          },
+          "activity_action": {
+            "type": "string",
+            "enum": [
+              "schedule"
+            ],
+            "description": "Action related to the activity type property."
+          },
+          "channels": {
+            "type": "object",
+            "description": "Determines in which communication channel to send the message"
+          },
+          "message_text": {
+            "type": "object",
+            "properties": {
+              "body": {
+                "type": "string",
+                "description": "The main content of the message."
+              },
+              "subject": {
+                "type": "string",
+                "description": "A concise summary of the message content."
+              }
+            }
+          },
+          "cta_button_text": {
+            "type": "string",
+            "description": "The text for the Email action button"
+          },
+          "link_url_params": {
+            "type": "object",
+            "description": "The parameters for the Email action button or SMS link."
+          }
+        },
+        "required": [
+          "client_uid",
+          "activity_type",
+          "activity_action",
+          "channels"
+        ]
+      }
+    }
+  },
+  "paths": {
+    "/v3/messaging/activity_message": {
+      "post": {
+        "operationId": "sendActivityMessage",
+        "summary": "Send activity message api",
+        "parameters": [],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RequestDto"
```

Comment:
TBH, there is no benefit in using a schema where it is used only once. I would take the schema directly into the endpoint definition. It will be easier to read. Same apply for the response